lib/modules/datasource/azure-bicep-resource/index.ts

```diff
@@ -35,54 +44,30 @@ export class AzureBicepResourceDatasource extends Datasource {
     const { packageName } = getReleasesConfig;
 
     const resourceVersionIndex = await this.getResourceVersionIndex();
-    const versions = resourceVersionIndex[packageName.toLowerCase()];
 
-    if (!versions) {
+    const versions = resourceVersionIndex[packageName.toLowerCase()];
+    if (!versions?.length) {
       return null;
     }
 
-    const firstSlashIndex = packageName.indexOf('/');
-    const namespaceProvider = packageName
-      .slice(0, firstSlashIndex)
-      .toLowerCase();
-    const type = packageName.slice(firstSlashIndex + 1).toLowerCase();
-
-    return {
-      releases: versions.map((version) => ({
-        version,
-        changelogUrl: `https://learn.microsoft.com/en-us/azure/templates/${namespaceProvider}/change-log/${type}#${version}`,
-      })),
-    };
+    const changelogUrl = this.getChangelogUrl(packageName);
+    const releases = versions.map((version) => ({
+      version,
+      changelogUrl: `${changelogUrl}#${version}`,
+    }));
+    return { releases };
   }
 
   @cache({
     namespace: `datasource-${AzureBicepResourceDatasource.id}`,
     key: 'getResourceVersionIndex',
     ttlMinutes: 24 * 60,
   })
-  async getResourceVersionIndex(): Promise<Record<string, string[]>> {
-    const res = await this.getBicepTypeIndex();
-
-    const releaseMap = new Map<string, string[]>();
-
-    for (const resourceReference of Object.keys(res.Resources)) {
-      const [type, version] = resourceReference.toLowerCase().split('@', 2);
-      const versions = releaseMap.get(type) ?? [];
-      versions.push(version);
-      releaseMap.set(type, versions);
-    }
-
-    for (const functionResource of Object.entries(res.Functions)) {
-      const [type, versionMap] = functionResource;
-      const versions = Object.keys(versionMap);
-      releaseMap.set(type, versions);
-    }
-
-    return Object.fromEntries(releaseMap);
-  }
-
-  private async getBicepTypeIndex(): Promise<BicepTypeIndex> {
-    const res = await this.http.getJson(BICEP_TYPES_INDEX_URL, BicepTypeIndex);
-    return res.body;
+  async getResourceVersionIndex(): Promise<BicepResourceVersionIndex> {
+    const { body: resourceVersionIndex } = await this.http.getJson(
+      BICEP_TYPES_INDEX_URL,
+      BicepResourceVersionIndex
+    );
+    return resourceVersionIndex;
```

Comment:
```suggestion
    const { body } = await this.http.getJson(
      BICEP_TYPES_INDEX_URL,
      BicepResourceVersionIndex
    );
    return body;
```
i don't see any benefits to rename body here.